src/content/docs/network-performance-monitoring/troubleshooting/understanding-snmp-utilization-calculations.mdx

```diff
@@ -106,9 +106,27 @@ You have questions about various results calculated by the `ktranslate` network
 
     Interface throughput uses the value of either [ifHCInOctets](https://oid-rep.orange-labs.fr/get/1.3.6.1.2.1.31.1.1.1.6) (receive) or [ifHCOutOctets](https://oid-rep.orange-labs.fr/get/1.3.6.1.2.1.31.1.1.1.10) (transmit).
 
-    It's important to note that there's a difference between utilization, which is typically represented as bits (lowercase `b`), and throughput, which many tools display in bytes (uppercase `B`). The raw measurement in SNMP is counting octets, which in this context would be synonymous with bytes.
+    It's important to note exactly what you are looking for when querying for throughput.
```

Comment:
@thezackm What are some examples of what they could be querying for? How is it relevant to the bullet points about octets/bytes/bits?

Reply:
@ally-sassman
throughput is a measurement of usage on a link, which is represented by either octets, bytes, or bits